batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/constant/StudentConstant.java

```diff
@@ -0,0 +1,17 @@
+package com.lftechnology.batch7crud.constant;
+
+public class StudentConstant {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)